core/storage/collection/gae_models.py

```diff
@@ -328,6 +327,26 @@ def convert_to_valid_dict(model_dict):
             model_dict['voice_artist_ids'] = model_dict['translator_ids']
             model_dict['translator_ids'] = []
 
+        # We need to remove pseudonymous IDs from all the fields that contain
+        # user IDs.
+        model_dict['owner_ids'] = [
```

Comment:
I think you can make this a for loop over a tuple, rather than repeating the logic.